test/utils/fixture/fixture.go

```diff
@@ -128,17 +134,19 @@ func Skipf(ft *Fixture, format string, args ...interface{}) {
 
 func Cooldown(ft *Fixture) {
 	if len(ft.InitialNRTList.Items) > 0 {
-		ginkgo.By("cooldown by verifying NRTs data is settled to the initial state")
-		intwait.With(ft.Client).Interval(5*time.Second).Timeout(1*time.Minute).ForNodeResourceTopologiesEqualTo(context.TODO(), &ft.InitialNRTList, intwait.NRTIgnoreNothing)
+		interval := 5 * time.Second
+		ginkgo.By(fmt.Sprintf("cooldown by verifying NRTs data is settled to the initial state (interval=%v timeout=%v)", interval, settleTimeout))
+		intwait.With(ft.Client).Interval(interval).Timeout(settleTimeout).ForNodeResourceTopologiesEqualTo(context.TODO(), &ft.InitialNRTList, intwait.NRTIgnoreNothing)
+		// TODO: shall we wait for the data to settle?
```

Comment:
I don't think we need this TODO because we're already waiting enough time for the NRT to equal to initial.

Reply:
sounds good. Removing.